opentreemap/treemap/templatetags/form_extras.py

```diff
@@ -377,12 +377,10 @@ def _field_value(model, field_name, data_type):
 
         if field_value is None:
             display_val = None
-        elif data_type == 'date' and model.instance:
-            display_val = dateformat.format(field_value,
-                                            model.instance.short_date_format)
-        elif data_type == 'date':
-            display_val = dateformat.format(field_value,
-                                            settings.SHORT_DATE_FORMAT)
+        elif data_type.startswith('date'):
```

Comment:
Is the change from `==` to `startswith` meant to catch both "date" and "datetime"?